cli/catalog-api-v1/README.md

```diff
@@ -34,9 +34,15 @@ Mid-term updating the client library and proposing it as a PR will be done by au
 [mitmproxy](https://mitmproxy.org/) can be used to debug requests and responses from the Catalog API.
 
 1. Start the interface and leave it running in a separate terminal:
-
-        mitmproxy
-
+    ```
+    nix run github:flox/nixpkgs/stable.20240203#mitmproxy
```

Comment:
To avoid the shell interpreting the hash as a comment:
```suggestion
    nix run 'github:flox/nixpkgs/stable.20240203#mitmproxy'
```

Reply:
the `#` inside a word is not starting a comment in either zsh or bash
`*`, `?`, `&` in other flakerefs are another story

Reply:
Sorry, I mean extended globbing in zsh:
```
% nix run github:flox/nixpkgs/stable.20240203#mitmproxy
zsh: no matches found: github:flox/nixpkgs/stable.20240203#mitmproxy
```